Sources/NIO/BlockingIOThreadPool.swift

```diff
@@ -75,6 +75,8 @@ public final class BlockingIOThreadPool {
 
     /// Submit a `WorkItem` to process.
     ///
+    /// - note: This is a low-level method, in most cases the `submit` method should be used.
```

Comment:
This should presumably be `runIfActive`, not `submit`.

Reply:
fixed, thanks